certified-connectors/ascentcloudterritoryplanner/README.md

```diff
@@ -0,0 +1,21 @@
+# Ascent Cloud Territory Planner Connector
+
+The Territory Planner connector from Ascent Cloud allows integration with Ascent Cloud's Territory Planner solution.
+
+The connector can be used in PowerApp and Power Automate to trigger territory plan data fetching.
+
+## Pre-requisites
+
+You will need:
+ * Active Microsoft Power Automate or Power Apps account
+ * A Territory Planner license and application subscription
+ * The organization Id and API token linked to you Territory Planner application subscription and organization
```

Comment:
@simongroom thank you so much for your submission. All files should be free of grammatical errors and typos. in this paragraph you have "...API token linked to you Territory Planner..." please change to "...API token linked to **your** Territory Planner...".
Your readme file is missing some required fields. please see our [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) and for code raw view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md). Please include all required sections.